code-challenges.test.js

```diff
@@ -25,27 +25,59 @@ const numbersArray2 = [24, 27, 30, 33, 36]
-// Expected output: "0 is divisible by three"
-const object3 = { number: -7 }
-// Expected output: "-7 is not divisible by three"
+describe("divisibleByThree", () => {
+    const object1 = { number: 15 }
+    // Expected output: "15 is divisible by three"
+    const object2 = { number: 0 }
+    // Expected output: "0 is divisible by three"
+    const object3 = { number: -7 }
+    // Expected output: "-7 is not divisible by three"
+    it("takes a object as an argument and decides if the number inside it is evenly divisible by three or not.", ()=> {
+        expect(divisibleByThree(object1)).toEqual("15 is divisible by three")
+        expect(divisibleByThree(object2)).toEqual("0 is divisible by three")
+        expect(divisibleByThree(object3)).toEqual("-7 is not divisible by three")
+    })
+})
 
 // b) Create the function that makes the test pass.
 
 // Pseudo code:
+//input: object
+//output: a string that says the number is divisible by three
+const divisibleByThree = (object) => {
+    // set if/else conditions to check truthy or falsely
+    if(object.number % 3 === 0){
+        // return the string if the if condition is true
+        return `${object.number} is divisible by three`
+    }else{
+        // return the string if the if condition is false
+        return `${object.number} is not divisible by three`
+    }
+}
+
 
 // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
-// Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
-const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
-// Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
+describe("toCapitalizedWords", () =>{
+    const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+    // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
+    const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
+    // Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
 
-// b) Create the function that makes the test pass.
+    it("takes in an array of words and returns an array with all the words capitalized." , () =>{
+        expect(toCapitalizedWords(randomNouns1)).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+        expect(toCapitalizedWords(randomNouns2)).toEqual(["Temperature", "Database", "Chopsticks", "Mango"])
+    })
+})
 
-// Pseudo code:
\ No newline at end of file
+// b) Create the function that makes the test pass.
+// Pseudo code:
+//input: array
+//output: array with capitalized words
+//define a function that takes in one parameter
+const toCapitalizedWords = (array)=>{
+    //use .map() to return the same length of an array and change each value to capitalized words
```

Comment:
I would expand on this a little more. Here's an example:
- use .map to iterate
- each iteration capitalizes the first letter of each value and adds the rest of the value to the capitalized letter